### DISCUSSION TITLE
When a tex file is in `tex` or `LaTeX-expl3`, the button of view pdf disappears and snippet of synctex from cursor doesn't work.

### DISCUSSION THREAD

**Discussion by syvshc** [James-Yu/LaTeX-Workshop#3170] [Q&A] [answered]
## Preliminary questions [Required]

When I switch language to `tex` or `LaTeX-expl3`, the button of `view pdf` disappears and snippet of `synctex from cursor` doesn't work. 

### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]

A clear and concise description of what the bug is

When I switch the format into `TeX` or `LaTeX-expl3`, there is no button of `View LaTeX PDF File`. 

### To Reproduce

1. Open a `.tex` file
2. Compile it with `-synctex=1` option
3. switch the format between `LaTeX` and `LaTeX-expl3` and see difference. 

### Expected behavior

My expected behavior is that "view pdf" button is still there and snippet of synctex works

## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

[lw.log](https://github.com/James-Yu/LaTeX-Workshop/files/8150575/lw.log)

### LaTeX Workshop Output [Required]

```
To access the log, click the 'TeX' icon on the Activity Bar on the left side, select 'View Log Messages', then select 'View LaTeX Workshop extension log'.

[Paste the log here]
```
It is no business about compiling, just SyncTeX. 

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools' -> 'Console'. Paste anything suspicious.

[Paste the log here]
```
![image](https://user-images.githubusercontent.com/38098591/155923589-c9cfc763-308b-4e03-ae9d-8fa832af5ee3.png)

## Screenshots

I recorded a video to describe my problem, hope it is clear

https://user-images.githubusercontent.com/38098591/155933189-bde21561-8b0f-48c2-ae72-815ad2fffbdd.mp4

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: Arch linux 5.16.11
 - VS Code version: 1.64.2
 - Extension version: 8.23.0
 - TeX distribution version: TeXLive 2021

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

**Comment by James-Yu** [comment]
#3160

**Comment by James-Yu** [comment]
Please confirm the issue with built-in viewer, and create a new issue if reproduced.

**Reply by syvshc** [reply]
I found that I just have to substitute the `.mp4` file, other things are no need to change.

**Comment by jlelong** [comment, marked as answer]
This is a bug. I have just converted the discussion back to an issue #3171
